pending-documentation.md

```diff
@@ -1213,6 +1391,72 @@ Here we know that `foo(I +Int 1)` will return an Int, but the return sort of
 `foo` is `Exp`. So we project the result into the `Int` sort so that it can
 be placed as the child of a `bar`.
 
+### `owise` and `priority` attributes.
+
+Sometimes, it is simply not convenient to explicitly describe every
+single negative case under which a rule should **not** apply. Instead,
+we simply wish to say that a rule should only apply after some other set of
+rules have been tried. K introduces two different attributes that can be
+added to rules which will automatically generate the necessary matching
+conditions in a manner which is performant for concrete execution (indeed,
+it generally outperforms during concrete execution code where the conditions
+are written explicitly).
+
+The first is the `owise` attribute. Very roughly, rules without an attribute
+indicating their priority apply first, followed by rules with the `owise`
+attribute only if all the other rules have been tried and failed. For example,
+consider the following function:
+
+```k
+syntax Int ::= foo(Int) [function]
+rule foo(0) => 0
+rule foo(_) => 1 [owise]
+```
+
+Here `foo(0)` is defined explicitly as 0. Any other integer yields the integer
+1. In particular, the second rule above will only be tried after the first
+rule has been shown not to apply.
+
+This is because the first rule has a lower number assigned for its priority
+than the second rule. In practice, each rule in your semantics is implicitly
+or explicitly assigned a numerical priority. Counterintuitively, numbers which
+are lower correspond to **higher** priority, i.e., they will be tried first.
+
```

Comment:
I would remove the "Counterintuitively", and instead say something to the effect that "rules are tried in increasing order of priority, starting at priority `0` and ending at priority `200`". It's only counterintuitive when you think of the number as how "prioritous" a rule is, instead of "where the rule goes in teh priority list".